### PR TITLE
libpanel 1.2.0 (new formula)

### PR DESCRIPTION
**Pull request by taozuhong**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
`docs/meson.build:7:13: ERROR: Program 'gi-docgen' not found or not executable`

File: Formula/libpanel.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      #include <libpanel.h>
+
+      int main(int argc, char *argv[]) {
+        uint major = panel_get_major_version();
+        return 0;
+      }
+    EOS
+    flags = shell_output("#{Formula["pkg-config"].opt_bin}/pkg-config --cflags --libs libpanel-1").strip.split
+    flags += shell_output("#{Formula["pkg-config"].opt_bin}/pkg-config --cflags --libs libadwaita-1").strip.split
+    system ENV.cc, "test.c", "-o", "test", *flags
```
Comment (SMillerDev) [inline Formula/libpanel.rb:35]: Does the `--help` do anything?

Comment (taozuhong) [reply Formula/libpanel.rb:35]: Do nothing, just copy from other formula, just keep for respect.

Comment (SMillerDev) [reply Formula/libpanel.rb:35]: Please don't add useless flags

Comment (taozuhong) [reply Formula/libpanel.rb:35]: Got it, thanks.

**Comment by chenrui333** [comment]
@taozuhong thanks for the PR, looks good to me, can you squash all commits into one with commit msg as `libpanel 1.2.0 (new formula)`

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**